QuizWhois.Api/Controllers/QuestionRatingController.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using QuizWhois.Common.Models;
+using QuizWhois.Domain.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace QuizWhois.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class QuestionRatingController : Controller
+    {
+        private readonly IQuestionRatingService _questionRatingService;
+
+        public QuestionRatingController(IQuestionRatingService questionRatingService)
+        {
+            _questionRatingService = questionRatingService;
+        }
+
+        [HttpPost]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult Post(QuestionModel questionModel, long userId, uint rating)
+        {
+            var addedRating = _questionRatingService.AddRating(questionModel, userId, rating);
+            return Ok(addedRating);
+        }
+
+        [HttpPut]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public IActionResult Update(QuestionModel questionModel, long userId, uint rating)
```

Comment:
I am not sure it is needed to change such a small thing as a mark, seems excessive.